tuist/tuist
# Remove unnecessary Copy Executables build phase.

Resolves https://github.com/tuist/tuist/issues/7063.

### Short description 📝

Since the dependencies on this phase are already captured in the Copy Products build phase, there's no need for this phase.

### How to test the changes locally 🧐

Download the project from the issue, and observe that the issue can no longer be reproduced with this change.

### Contributor checklist ✅


### Reviewer checklist ✅
